### DISCUSSION TITLE
How to set power on password ?

### DISCUSSION THREAD

**Discussion by goopaga** [egzumer/uvk5-chirp-driver#13] [Q&A] [answered]
Hi,
I've compiled from the main branch as of today (28/12/2023) with the "ENABLE_PWRON_PASSWORD" set to 1.
Flashed on the radio, Full Reset of the radio.

With CHIRP Next (20231220) + Egzumer Module (27/12/2023) I'm reading the radio settings and I can see the "power on password" capability in the browser\BUILD_OPTIONS view:

![image](https://github.com/egzumer/uvk5-chirp-driver/assets/96391761/91754c24-c73b-43f5-9bd2-b1ef1706e8e8)

However I cannot figure out how to set the password as there are no password related options in the settings page:

![image](https://github.com/egzumer/uvk5-chirp-driver/assets/96391761/cd3f6a23-e817-40da-9a22-4743ba71b6b8)

AFAIK the original way to set a password was through the CPS software, however it's not anymore compatible with the settings structure of the OEFW versions.

Thanks for your thougths.

**Comment by egzumer** [comment]
Password was not implemented in chirp. It is not available right now.

**Reply by goopaga** [reply]
Understood, thanks.

As far as you know, is there any other way right now to set a password ?

**Reply by egzumer** [reply]
You can still use CPS but ot will overwrite some other options, that you will have to setup again.

**Comment by egzumer** [comment, marked as answer]
Use current version of the driver, there is a password setting in advanced settings

**Reply by Boris8800** [reply]
Can you add password option in the menu please 🙏.
